susemanager-utils/testing/docker/scripts/reportdb_schema_idempotency_test_pgsql.py

```diff
@@ -171,6 +172,7 @@ def create_fake_migration_path(schema_path, new_version, pr_file=None, version=N
 
 def run_command(command):
     """Run a shell command"""
+    # print(f"Run: {command}")
```

Comment:
[nit]
Can we remove this? Either we can print the command, or remove the comment.